install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+: '
+This script is used when there are multiple modular configurations available for METEOR.
+It is recommended to combine it with the exec= command of the Odemis.desktop file.
+However it could be used stand-alone if it is required.
+The syntax to run this script -> odemis-select-mic-start [search_dir] [search_pat].
```

Comment:
is it possible to run `odemis-select-mic-start --help` to get an explanation of the input arguments?